packages/fleather/lib/src/widgets/editor_selection_delegate_mixin.dart

```diff
@@ -81,7 +81,9 @@ mixin RawEditorStateSelectionDelegateMixin on EditorState
 
   @override
   void hideToolbar([bool hideHandles = true]) {
-    if (selectionOverlay?.toolbarIsVisible == true) {
+    if (hideHandles && selectionOverlay?.handlesAreVisible == true) {
```

Comment:
Do we need to the check `handlesAreVisible` since `hide()` already handles `handles == null`?